core/domain/wipeout_service_test.py

```diff
@@ -163,6 +163,11 @@ def setUp(self):
         self.user_1_id = self.get_user_id_from_email(self.USER_1_EMAIL)
         self.user_2_id = self.get_user_id_from_email(self.USER_2_EMAIL)
         self.user_1_gae_id = self.get_gae_id_from_email(self.USER_1_EMAIL)
+        user_services.set_user_pin(self.user_1_id, '12345')
+        user_services.create_new_profile(
```

Comment:
Can we test that the profile is also deleted in the tests?

Reply:
Done now.